components/org.wso2.carbon.identity.oauth.scope.endpoint/pom.xml

```diff
@@ -157,19 +152,14 @@
         </dependency>
         <dependency>
             <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
-            <artifactId>org.wso2.carbon.identity.oauth</artifactId>
+            <artifactId>org.wso2.carbon.identity.api.server.oauth.scope</artifactId>
             <exclusions>
                 <exclusion>
-                    <groupId>javax.ws.rs</groupId>
-                    <artifactId>jsr311-api</artifactId>
+                    <groupId>*</groupId>
```

Comment:
Why do we have * here?

Reply:
We want to exclude all the transitive dependencies from the new module being packed inside the lib of the webapp.